core/src/main/java/org/apache/carbondata/core/cache/CarbonLRUCache.java

```diff
@@ -95,9 +98,17 @@ public CarbonLRUCache(String propertyName, String defaultPropertyName) {
    * initialize lru cache
    */
   private void initCache() {
+    // get duration for cache expiration if configured
+    long duration = CarbonCommonConstants.CARBON_LRU_CACHE_EXPIRATION_DURATION_IN_MINUTES_DEFAULT;
+    String timeBasedExpiration = CarbonProperties.getInstance()
+        .getProperty(CarbonCommonConstants.CARBON_LRU_CACHE_EXPIRATION_DURATION_IN_MINUTES);
+    if (null != timeBasedExpiration) {
+      duration = Long.parseLong(timeBasedExpiration);
```

Comment:
forget to validate the content?

Reply:
Configuration is already validated in `core/src/main/java/org/apache/carbondata/core/util/CarbonProperties.java`. Please check